src/main/java/com/linkedin/kmf/services/ProduceService.java

```diff
@@ -64,7 +64,7 @@
   private final String _name;
   private final ProduceMetrics _sensors;
   private KMBaseProducer _producer;
-  private KMPartitioner _partitioner;
+  private final KMPartitioner _partitioner;
```

Comment:
This seems outside of rebranding.  Would suggest that it gets split out into another commit for history hygiene.

Reply:
sgtm